### ISSUE TITLE
Automate install of Kai client distributed as a zip file

### ISSUE THREAD

**Issue by dymurray** [konveyor/kai#342]

**Comment by midays** [comment]
Hi @dymurray, just to be sure cause I got a little confused, by kai client you mean the kai vscode plugin, correct ? if not could you please elaborate ?

**Comment by dymurray** [comment]
@midays no, this is referring to the python client that is being delivered by the end of this sprint. To be honest, this work might be premature until we finish https://github.com/konveyor/kai/issues/330.

For now, I think we would want to ensure we understand https://github.com/konveyor/kai/pull/321 so that we know what deps we need to support pyinstaller in our automation
